board/app/controllers/user_controller.php

```diff
@@ -1,92 +1,129 @@
 <?php
 
-class UserController extends AppController{
+class UserController extends AppController 
+{
+
+const LOGIN_PAGE = 'login';
+const HOME_PAGE = 'home';
+const REGISTER = 'register';
+const REGISTER_END = 'register_end';
+const EDIT = 'edit';
+const EDIT_END = 'edit_end';
 
     public function register() 
     {
-        if (is_logged_in()) 
-        {
-            redirect(url('user/home'));
+        if (is_logged_in()) {
+            redirect(url('comment/home'));
```

Comment:
Are the lines15~17 still necessary? Since UserController inherits from AppController (where you called on its beforeFilter to check if user is_logged_in()), is_logged_in() is already automatically called for all that inherits from AppController. :bow: